tests/src/com.google.protobuf/protobuf-java-util/3.21.12/src/test/resources/META-INF/native-image/protobuf-java-util-test-metadata/reflect-config.json

```diff
@@ -0,0 +1,6379 @@
+[
```

Comment:
Hey - why is this file necessary?

Reply:
- This file is all GraalVM metadata needed by the test class itself, I haven't tried to simplify it. According to the unit test I did on the ShardingSphere side, protobuf requires a lot of reflection.

Reply:
There are a lot of references to classes in there which are not test-specific, like `"name": "com.google.protobuf.Any"` or `"name": "com.google.protobuf.BoolValue"`. My problem is that this could lead to the test passing even if the metadata in the `metadata/` folder is not complete. Can you try to get rid of that file or at least condense it so that it only references classes from your test?

Reply:
- I made the following changes in https://github.com/linghengqian/graalvm-trace-metadata-smoketest/commit/f9de2103ebb52208ccb28fa0098a3322d0970a60 and put them into the current PR.
- Replace most of `"typeReachable": "com.google.protobuf.util.JsonFormatTest"` with `"typeReachable": "com.google.protobuf.GeneratedMessageV3"`.
- Remove most entries for `"typeReachable": "com.google.protobuf.util.DurationsTest"` and `"typeReachable": "com.google.protobuf.util.FieldMaskTreeTest"`.
- The reason why the file appears to have many lines is because each entry is very large, making the line number bloated.
- ![image](https://user-images.githubusercontent.com/20187731/212858351-59ab9eb1-daf2-4be1-97e4-668568e868d3.png)